packages/typescript-estree/src/create-program/getScriptKind.ts

```diff
@@ -1,6 +1,32 @@
 import path from 'path';
 import * as ts from 'typescript';
 
+function isNodeNextModuleExtension(filePath: string): boolean {
```

Comment:
we don't need to put this within `typescript-estree` as t's not used here and won't ever be used here - this can just live directly in our plugin - I think it can probably live directly within the rule's file.

I'm don't think we really want to expose this - I don't think many other plugins (if any) are going to be wanting this sort of thing.

Reply:
Sure 👍